src/AuthModule/Components/Register/Register.tsx

```diff
@@ -1,7 +1,432 @@
+                      color: "#eb5148",
+                      textDecoration: "none",
+                      fontWeight: 600,
+                      marginLeft: 5,
+                    }}
+                  >
+                    Login here !
+                  </Link>
+                </Typography>
+              </Box>
+              <Box
+                component="form"
+                noValidate
+                sx={{ mt: 1 }}
+                onSubmit={handleSubmit(handleRegister)}
+              >
+                <Box
+                sx={{ display: "flex", flexDirection:'column', alignItems: "center" }}
+                >
+                  <Box sx={{width:'100px',height:'100px'}}><img src={userImage} alt="" style={{width:'100%'}} /></Box>
+                <Button
+                
+                  component="label"
+                  role={undefined}
+                  variant="outlined"
+                  tabIndex={-1}
+                  // startIcon={<CloudUploadIcon />}
+                  sx={{mb:2}}
+                >
+                  Upload image
+                  <VisuallyHiddenInput type="file" 
+                  {...register("profileImage", { required: "Image is required" })}
+                  onChange={(e)=>setUserImage(URL.createObjectURL(e?.target?.files[0]))}
+                  />
+                </Button>
+                {errors.profileImage && (
+                    <Typography variant="body2" sx={{ color: "red" }}>
+                      {errors?.profileImage?.message}
+                    </Typography>
+                  )}
+                </Box>
+
+                <Box>
+                  <Typography sx={{ mt: 1 }}>User Name</Typography>
+                  <TextField
+                    // sx={{bgcolor:'#F5F6F8'}}
+                    placeholder="Please type here ..."
+                    margin="normal"
+                    required
+                    fullWidth
+                    id="username"
+                    // label="User Name"
+                    autoComplete="username"
+                    autoFocus
+                    {...register("userName", userNameValidation)}
+                  />
+                  {errors.username && (
+                    <Typography variant="body2" sx={{ color: "red" }}>
+                      {errors?.userName?.message}
+                    </Typography>
+                  )}
+                </Box>
+
+                <Box
+                  sx={{
+                    display: "flex",
+                    justifyContent: "space-between",
+                  }}
+                >
+                  <Box>
+                    <Typography sx={{ mt: 1 }}>Phone Number</Typography>
+                    <TextField
+                      // sx={{bgcolor:'#F5F6F8'}}
+                      placeholder="Please type here ..."
+                      margin="normal"
+                      required
+                      fullWidth
+                      id="phoneNumber"
```

Comment:
We can add type="tel" here, this will enable people using mobile phones to access keypad directly.